server/src/main/java/org/opensearch/cluster/service/ClusterManagerTaskThrottler.java

```diff
@@ -50,6 +51,9 @@ public class ClusterManagerTaskThrottler implements TaskBatcherListener {
     private final ConcurrentMap<String, Long> tasksCount;
     private final ConcurrentMap<String, Long> tasksThreshold;
     private final Supplier<Version> minNodeVersionSupplier;
+    // Once all nodes are higher than 2.5.0 version, then only it will start throttling.
+    // Needed for static threshold settings.
```

Comment:
Also add ->  During upgrade, it will wait for all older version nodes to leave the cluster and then only start throttling.

Reply:
Ack.